[PL-51921] DEL 83300 Release Notes (#7097)
harness/developer-hub: release-notes/delegate.md

## June 2024

> ### Version 24.06.83300 <!--  June 24, 2024 -->

Reviewer: ```suggestion
### Version 24.06.83303 <!--  June 24, 2024 -->
```

#### Fixed issues

> - Kubernetes services was created during the startup of the delegate, causing the IP pool to be exhausted for NAB. The delegate has been updated to prevent the creation of Kubernetes services upon startup, resolving the issue with IP pool exhaustion. (PL-51550)

Reviewer: ```suggestion
- Kubernetes services were created during the startup of the delegate, causing the IP pool to be exhausted for NAB. The delegate has been updated to prevent the creation of Kubernetes services upon startup, resolving the issue with IP pool exhaustion. (PL-51550)
```

> - Delegate was running out of memory due to frequent connectivity checks. Optimized the connectivity check process to reduce memory usage, preventing the delegate from running out of memory. (PL-51418, ZD-63705)

Reviewer: ```suggestion
- Delegates were running out of memory due to frequent connectivity checks. Optimized the connectivity check process to reduce memory usage, preventing the delegate from running out of memory. (PL-51418, ZD-63705)
```
